envfile.sample

```diff
@@ -5,7 +5,8 @@ BIND_PORT=8000
 CAUSEMOS_DEBUG=true
```

Comment:
Should `CAUSEMOS_DEBUG` be set to false by default in this file as well?

I'm not sure myself: I had not used individual nested project .env files (such as `api/.env`) before. I saw top-level `.env` files were ignored by git in our config, but nested ones aren't.

I do checked that dmc/ has a different DEBUG flag on, and still points to full causemos url. Maybe no change for these values are needed, but it is a bit hard to follow (maybe not part of this work, but something to track).

What do you think?

Reply:
we are removing the contents of those sub .env files so they shouldn't be an issue after that change is merged.

Reply:
Ah- forgot about that very recent change in dev.